custom URL scheme normative language and fixed non-normative example (#253)
openid/OpenID4VP: openid-4-verifiable-presentations-1_0.md

The Verifier has the choice of the following mechanisms to invoke a Wallet:

> - Custom URL scheme as an `authorization_endpoint` using `openid4vp://` as defined in (#openid4vp-custom-url-scheme)

Reviewer: I'm worried this might make it look like only `openid4vp://` can be used, whereas, for example, 18013-7 iso mdl defines it's own custom scheme, `mdoc-openid4vp://`

Author: since there is space for more than a single custom url scheme using openid4vp, I am wondering if it would be an ARRAY instead of a string

while about the normative MUST, we might say that at least openid4vp:// must be included

Reviewer 2: ```suggestion
- Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-profile))
```
+1 to Joseph's concern. please keep it as-is, or suggest a text that clarifies that openid4vp:// is not the only scheme that can be used.

> - Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-custom-url-scheme))

Author: ```suggestion
- Custom URL scheme as an `authorization_endpoint` using `openid4vp://` as defined in (#openid4vp-custom-url-scheme))
```

- Domain-bound Universal Links/App link as an `authorization_endpoint`
- no specific `authorization_endpoint`, user scanning a QR code with Authorization Request using a manually opened Wallet, instead of an arbitrary camera application on a user-device (neither custom URL scheme nor Universal/App link is used)

# Wallet Metadata (Authorization Server Metadata) {#as_metadata_parameters}

This specification defines how the Verifier can determine Credential formats, proof types and algorithms supported by the Wallet to be used in a protocol exchange.

## Authorization Endpoint Custom URL Scheme {#openid4vp-custom-url-scheme}

The custom URL scheme representing the Authorization Endpoint defined in this specification is `openid4vp`.

> Implementations that use the custom URL scheme defined in this specification MUST configure their `authorization_endpoint` metadata parameter using the value `openid4vp://`, presenting the custom URL scheme along with the hier-part and separated by a colon, as defined in [@!RFC3986]].

Author: ```suggestion
Implementations that use the custom URL scheme defined in this specification MUST configure their `authorization_endpoint` metadata parameter using the value `openid4vp://`, presenting the custom URL scheme along with the hier-part and separated by a colon, as defined in [@!RFC3986].
```

> Implementations that use the custom URL scheme defined in this specification MUST configure their `authorization_endpoint` metadata parameter the value `openid4vp://`, presenting the custom URL scheme along with the hier-part and separated by a colon, as defined in [@!RFC3986]].

Author: ```suggestion
Implementations that use the custom URL scheme defined in this specification MUST configure their `authorization_endpoint` metadata parameter using the value `openid4vp://`, presenting the custom URL scheme along with the hier-part and separated by a colon, as defined in [@!RFC3986]].
```

> ## Authorization Endpoint Custom URL Scheme {#openid4vp-custom-url-scheme}
> 
> The custom URL scheme representing the Authorization Endpoint defined in this specification is `openid4vp`.
> Implementations that use the custom URL scheme defined in this specification MUST configure their `authorization_endpoint` metadata parameter using the value `openid4vp://`, presenting the custom URL scheme along with the hier-part and separated by a colon, as defined in [@!RFC3986].
> 
> 

Reviewer 2: I think there is a misunderstanding. openid4vp does not mandate supporting openid4vp. this specification only defines a static metadata config that can be used with openid4vp, but it is not mandatory to support it by any means.

```suggestion
## Authorization Endpoint and a Custom URL Scheme {#openid4vp-custom-url-scheme}

In this specification, the Authorization Endpoint can be a custom URL scheme.
```

- [JWT VC Presentation Profile](https://identity.foundation/jwt-vc-presentation-profile/)

> ### A Set of Static Configuration Values bound to `openid4vp://` {#openid4vp-profile-consd}

Author: ```suggestion
### A Set of Static Configuration Values bound to `openid4vp://` {#openid4vp-profile}
```

### A Set of Static Configuration Values bound to `openid4vp://` {#openid4vp-profile}

Below is a non-normative example of a set of static configuration values that can be used with `vp_token` parameter as a supported Response Type, bound to the `openid4vp` Authorization Endpoint in the form of custom URL scheme. 

```json
{

>   "authorization_endpoint": "openid4vp://",

Reviewer 2: we had a long debate about this and agreed that `openid4vp:` is a way to express this, not `openid4vp://`
```suggestion
  "authorization_endpoint": "openid4vp:",
```

Reviewer 2: we don't say `https://` in the spec, we just say `https` scheme
